src/ucai/core/databricks.py

```diff
@@ -211,6 +212,34 @@ def create_function(
             return self.client.functions.create(function_info)
         raise ValueError("Either function_info or sql_function_body should be provided.")
 
+    @override
+    def create_python_function(self, *, func: Callable, func_comment: str, catalog: str, schema: str) -> "FunctionInfo":
+        """
+        Create a UC function from a Python function.
+
+        Args:
+            func: The Python callable function to convert into a UDF.
```

Comment:
I think we should add an example and clarify what the requirements are.

Reply:
I was thinking the same thing :) Even my suggestion above is too ambiguous about how to use this. Since this API is actually doing quite a bit of work, I'm thinking of adding a few examples to the docstring just to show options that are available (good, good with comments, bad with no type hints, bad with wrong format docstrings)